### PR TITLE
api-docs: Add organization name to page header

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

Current API Explorer always displayed, "Backstage API Explorer". Updates to use the org name such as "My Company API Explorer", if defined, falling back to Backstage if not.

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4713-2wvwewjyxoly4furtdahcydgrflgzfi0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60387cb365f8171a37fb40d1

File: plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerLayout.tsx [hunk 1]
```diff
@@ -14,20 +14,25 @@
...
-      subtitle="Backstage API Explorer"
-      pageTitleOverride="APIs"
-    />
-    {children}
-  </Page>
-);
+export const ApiExplorerLayout = ({ children }: Props) => {
+  const configApi = useApi(configApiRef);
+  const generatedSubtitle = `${
+    configApi.getString('organization.name') ?? 'Backstage'
```
Comment (Rugvip) [inline plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerLayout.tsx:26]: Gonna need a `getOptionalString` here

**Review by Rugvip — APPROVED** [review]
Thanks! 👍 

Small fix needed before merge

File: plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerLayout.tsx (hunk 1 above)
Comment (adamdmharvey) [reply plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerLayout.tsx]: ah yes 👍

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4713-2wvwewjyxoly4furtdahcydgrflgzfi0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60387cb365f8171a37fb40d1

**Comment by adamdmharvey** [comment]
I tried to sign-off on these old commits.

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4713-2wvwewjyxoly4furtdahcydgrflgzfi0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60387cb365f8171a37fb40d1

**Pull request merged**